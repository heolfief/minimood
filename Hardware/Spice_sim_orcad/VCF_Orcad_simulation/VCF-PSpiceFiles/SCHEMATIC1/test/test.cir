** Profile: "SCHEMATIC1-test"  [ C:\Users\Julien\Documents\INSA EII\Semestre 8\ProjetPluri\minimood\Hardware\VCF_Orcad_simulation\vcf-pspicefiles\schematic1\test.sim ] 

** Creating circuit file "test.cir" 
** WARNING: THIS AUTOMATICALLY GENERATED FILE MAY BE OVERWRITTEN BY SUBSEQUENT SIMULATIONS

*Libraries: 
* Profile Libraries :
* Local Libraries :
* From [PSPICE NETLIST] section of C:\Users\Julien\AppData\Roaming\SPB_Data\cdssetup\OrCAD_PSpice\17.4.0\PSpice.ini file:
.lib "nom.lib" 

*Analysis directives: 
.TRAN  0 10m 0 1u 
.OPTIONS ADVCONV
.PROBE64 V(alias(*)) I(alias(*)) W(alias(*)) D(alias(*)) NOISE(alias(*)) 
.INC "..\SCHEMATIC1.net" 


.END
