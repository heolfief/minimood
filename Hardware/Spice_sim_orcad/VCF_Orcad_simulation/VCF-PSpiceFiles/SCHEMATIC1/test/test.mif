 lib=C:\Cadence\SPB_17.4\tools\PSpice\Library\ebipolar.lib, offset=7160, size=365
 lib=C:\Cadence\SPB_17.4\tools\PSpice\Library\breakout.lib, offset=9201, size=133
 lib=C:\Cadence\SPB_17.4\tools\PSpice\Library\microchip_opamp.lib, offset=2789, size=3092
