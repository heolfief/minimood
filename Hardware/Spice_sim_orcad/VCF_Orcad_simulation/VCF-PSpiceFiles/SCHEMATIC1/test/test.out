
**** 02/10/20 10:57:19 ******* PSpice 17.4.0 (Nov 2018) ******* ID# 0 ********

 ** Profile: "SCHEMATIC1-test"  [ C:\Users\Julien\Documents\INSA EII\Semestre 8\ProjetPluri\minimood\Hardware\VCF_Orcad_simulation\vc


 ****     CIRCUIT DESCRIPTION


******************************************************************************




** Creating circuit file "test.cir" 
** WARNING: THIS AUTOMATICALLY GENERATED FILE MAY BE OVERWRITTEN BY SUBSEQUENT SIMULATIONS

*Libraries: 
* Profile Libraries :
* Local Libraries :
* From [PSPICE NETLIST] section of C:\Users\Julien\AppData\Roaming\SPB_Data\cdssetup\OrCAD_PSpice\17.4.0\PSpice.ini file:
.lib "nom.lib" 

*Analysis directives: 
.TRAN  0 10m 0 1u 
.OPTIONS ADVCONV
.PROBE64 V(alias(*)) I(alias(*)) W(alias(*)) D(alias(*)) NOISE(alias(*)) 
.INC "..\SCHEMATIC1.net" 



**** INCLUDING SCHEMATIC1.net ****
* source VCF
V_V4         VCC 0 5Vdc
R_R15         N06499 N06395  1meg TC=0,0 
R_R13         N06339 N06345  10k TC=0,0 
C_C7         INPUT N06499  4.7u  TC=0,0 
R_R17         VCC N06669  100k TC=0,0 
X_R_RES         N06157 N06157 N06231 POT PARAMS: SET={Res} VALUE=10k
C_C6         N06395 N06219  10n  TC=0,0 
C_C5         N06219 N06231  10n  TC=0,0 
Q_Q2         N06219 N06345 0 BC547B 
C_C8         N06157 OUTPUTVCF  4.7u  TC=0,0 
R_R14         N06395 N06157  470k TC=0,0 
R_R16         0 N06499  100k TC=0,0 
R_R19         0 N06669  100k TC=0,0 
X_U3         N06669 N06395 VCC 0 N06157 MCP6001
X_R_CUT         0 N06339 VCC POT PARAMS: SET={Freq} VALUE=10K
R_R20         DAC_OUT N06345  10k TC=0,0 
V_V6         DAC_OUT 0 0
X_U4         N10926 N11078 VCC 0 N11095 MCP6001
R_R22         0 N10926  100k TC=0,0 
R_R21         VCC N10926  100k TC=0,0 
R_R23         N11078 N11095  10k TC=0,0 
R_R24         OUTPUTVCF N11078  3.9k TC=0,0 
C_C9         OUTPUT N11095  100n  TC=0,0 
R_R25         0 OUTPUT  1Meg TC=0,0 
V_V8         INPUT 0  AC 1
+SIN 0 .5 300 0 0 0
.PARAM  k=0 res=1 freq=.00001

**** RESUMING test.cir ****
.END

**** 02/10/20 10:57:19 ******* PSpice 17.4.0 (Nov 2018) ******* ID# 0 ********

 ** Profile: "SCHEMATIC1-test"  [ C:\Users\Julien\Documents\INSA EII\Semestre 8\ProjetPluri\minimood\Hardware\VCF_Orcad_simulation\vc


 ****     Diode MODEL PARAMETERS


******************************************************************************




               X_U3.DESD       X_U3.DL         X_U3.DN1        X_U4.DESD       
          IS    1.000000E-15    1.000000E-15    1.000000E-12    1.000000E-15 
          KF                                  146.000000E-18                 


               X_U4.DL         X_U4.DN1        
          IS    1.000000E-15    1.000000E-12 
          KF                  146.000000E-18 


**** 02/10/20 10:57:19 ******* PSpice 17.4.0 (Nov 2018) ******* ID# 0 ********

 ** Profile: "SCHEMATIC1-test"  [ C:\Users\Julien\Documents\INSA EII\Semestre 8\ProjetPluri\minimood\Hardware\VCF_Orcad_simulation\vc


 ****     BJT MODEL PARAMETERS


******************************************************************************




               BC547B          
               NPN             
       LEVEL    1            
          IS    7.049000E-15 
          BF  374.6          
          NF    1            
         VAF   62.79         
         IKF     .08157      
         ISE   68.000000E-15 
          NE    1.576        
          BR    1            
          NR    1            
         IKR    3.924        
         ISC   12.400000E-15 
          NC    1.835        
          NK     .4767       
         ISS    0            
          RE    0            
          RC     .9747       
         CJE   11.500000E-12 
         VJE     .5          
         MJE     .6715       
         CJC    5.250000E-12 
         VJC     .5697       
         MJC     .3147       
        XCJC    1            
         CJS    0            
         VJS     .75         
          TF  410.200000E-12 
         XTF   40.06         
         VTF   10            
         ITF    1.491        
          TR   10.000000E-09 
         XTB    1.5          
          KF    0            
          AF    1            
          CN    2.42         
           D     .87         


**** 02/10/20 10:57:19 ******* PSpice 17.4.0 (Nov 2018) ******* ID# 0 ********

 ** Profile: "SCHEMATIC1-test"  [ C:\Users\Julien\Documents\INSA EII\Semestre 8\ProjetPluri\minimood\Hardware\VCF_Orcad_simulation\vc


 ****     MOSFET MODEL PARAMETERS


******************************************************************************




               X_U3.NMI        X_U4.NMI        
               NMOS            NMOS            
       LEVEL    1               1            
           L  100.000000E-06  100.000000E-06 
           W  100.000000E-06  100.000000E-06 
         VTO    0               0            
          KP   20.000000E-06   20.000000E-06 
       GAMMA    0               0            
         PHI     .6              .6          
      LAMBDA    0               0            
          IS   10.000000E-15   10.000000E-15 
          JS    0               0            
          PB     .8              .8          
        PBSW     .8              .8          
          CJ    0               0            
        CJSW    0               0            
        CGSO    0               0            
        CGDO    0               0            
        CGBO    0               0            
         TOX    0               0            
          XJ    0               0            
       UCRIT   10.000000E+03   10.000000E+03 
      DIOMOD    1               1            
         VFB    0               0            
        LETA    0               0            
        WETA    0               0            
          U0    0               0            
        TEMP    0               0            
         VDD    5               5            
       XPART    0               0            


**** 02/10/20 10:57:19 ******* PSpice 17.4.0 (Nov 2018) ******* ID# 0 ********

 ** Profile: "SCHEMATIC1-test"  [ C:\Users\Julien\Documents\INSA EII\Semestre 8\ProjetPluri\minimood\Hardware\VCF_Orcad_simulation\vc


 ****     Resistor MODEL PARAMETERS


******************************************************************************




               X_U3.REXP       X_U4.REXP       
           R    1               1            
         TCE   10.1            10.1          


**** 02/10/20 10:57:19 ******* PSpice 17.4.0 (Nov 2018) ******* ID# 0 ********

 ** Profile: "SCHEMATIC1-test"  [ C:\Users\Julien\Documents\INSA EII\Semestre 8\ProjetPluri\minimood\Hardware\VCF_Orcad_simulation\vc


 ****     INITIAL TRANSIENT SOLUTION       TEMPERATURE =   27.000 DEG C


******************************************************************************



 NODE   VOLTAGE     NODE   VOLTAGE     NODE   VOLTAGE     NODE   VOLTAGE


(  VCC)    5.0000  (INPUT)    0.0000  (N06157)    3.5673 (N06219)     .0182     

(N06231)    3.5673 (N06339)    4.9999 (N06345)     .6563 (N06395)    2.4994     

(N06499)     .2272 (N06669)    2.5000 (N10926)    2.5000 (N11078)    2.4994     

(N11095)    2.4994 (OUTPUT)    0.0000 (DAC_OUT)    0.0000                       

(X_U3.10)    5.5000                   (X_U3.11)    5.3275                       

(X_U3.12)    5.3275                   (X_U3.13)    5.3000                       

(X_U3.14)    2.4994                   (X_U3.15)    2.1543                       

(X_U3.16)    -.3000                   (X_U3.20)     .5137                       

(X_U3.21)    -.5137                   (X_U3.26)-135.0E-06                       

(X_U3.27)-241.5E-06                   (X_U3.30)     .0022                       

(X_U3.31)  111.2500                   (X_U3.32) -130.2000                       

(X_U3.33)     .0022                   (X_U3.34)     .9391                       

(X_U3.35)     .9391                   (X_U3.37)     .9391                       

(X_U3.38)     .9391                   (X_U3.39)     .9391                       

(X_U3.50)     .0023                   (X_U3.51)     .0023                       

(X_U3.52) 229.7E-12                   (X_U3.55)    4.9992                       

(X_U3.56)     .0012                   (X_U3.57)    3.5673                       

(X_U3.61) 109.5E-06                   (X_U3.62) 58.20E-12                       

(X_U3.70) 1.000E-06                   (X_U3.71)    2.5000                       

(X_U3.96)    3.5673                   (X_U3.98) 1.079E-06                       

(X_U3.99) 2.272E-06                   (X_U4.10)    5.5000                       

(X_U4.11)    5.3275                   (X_U4.12)    5.3275                       

(X_U4.13)    5.3000                   (X_U4.14)    2.4994                       

(X_U4.15)    2.1543                   (X_U4.16)    -.3000                       

(X_U4.20)     .5137                   (X_U4.21)    -.5137                       

(X_U4.26)-135.0E-06                   (X_U4.27)-241.5E-06                       

(X_U4.30)-28.45E-06                   (X_U4.31)  111.2500                       

(X_U4.32) -130.2000                   (X_U4.33)-27.45E-06                       

(X_U4.34)    -.0117                   (X_U4.35)    -.0117                       

(X_U4.37)    -.0117                   (X_U4.38)    -.0117                       

(X_U4.39)    -.0117                   (X_U4.50)-648.5E-12                       

(X_U4.51)-65.50E-18                   (X_U4.52)-647.8E-12                       

(X_U4.55)    4.9993                   (X_U4.56)     .0012                       

(X_U4.57)    2.4994                   (X_U4.61) 109.5E-06                       

(X_U4.62) 58.20E-12                   (X_U4.70) 1.000E-06                       

(X_U4.71)    2.5000                   (X_U4.96)    2.4994                       

(X_U4.98)-305.5E-15                   (X_U4.99)-646.5E-15                       

(OUTPUTVCF)    2.4994                 




    VOLTAGE SOURCE CURRENTS
    NAME         CURRENT

    V_V4        -1.208E-03
    V_V6         6.563E-05
    V_V8         0.000E+00
    X_U3.V10     5.000E-05
    X_U3.V16     2.828E-12
    X_U3.V13     2.829E-12
    X_U4.V10     5.000E-05
    X_U4.V16     2.828E-12
    X_U4.V13     2.829E-12

    TOTAL POWER DISSIPATION   6.09E-03  WATTS



          JOB CONCLUDED

**** 02/10/20 10:57:19 ******* PSpice 17.4.0 (Nov 2018) ******* ID# 0 ********

 ** Profile: "SCHEMATIC1-test"  [ C:\Users\Julien\Documents\INSA EII\Semestre 8\ProjetPluri\minimood\Hardware\VCF_Orcad_simulation\vc


 ****     JOB STATISTICS SUMMARY


******************************************************************************



  License check-out time            =       98.11
  Total job time (using Solver 1)   =        1.77
